### PR TITLE
Make alt text editable by instructors

### PR DESCRIPTION
**Pull request by azimpradhan**

### CODE REVIEW

**Comment by azimpradhan** [comment]
@stvstnfrd, @caesar2164, please review

File: imagemodal/imagemodal.py [hunk 1]
```diff
@@ -107,6 +118,7 @@ def student_view(self, context=None):
                 'thumbnail_url': self.thumbnail_url or self.image_url,
                 'description': self.description,
                 'xblock_id': unicode(self.scope_ids.usage_id),
+                'alt_text': self.alt_text or self.display_name
```
Comment (caesar2164) [inline imagemodal/imagemodal.py]: yay for graceful degradation!

**Comment by caesar2164** [comment]
looks pretty good, can I get a demo?

File: imagemodal/imagemodal.py [hunk 2]
```diff
@@ -77,11 +77,22 @@ def workbench_scenarios():
...
     )
 
+    alt_text = String(
+        display_name=_('Alt Text'),
+        default='',
+        scope=Scope.settings,
+        help=_(
+            'This field allows you to add alternate or descriptive text'
+            'that pertains to your image.'
+        )
```
Comment (stvstnfrd) [inline imagemodal/imagemodal.py]: comma comma comma comma comma chameleon

File: imagemodal/imagemodal.py (hunk 1 above)
Comment (stvstnfrd) [inline imagemodal/imagemodal.py]: comma comma comma comma comma chameleon

**Comment by stvstnfrd** [comment]
@azimpradhan It looks like this PR contains files that shouldn't be part of this changeset. I don't think any of those JS files should have changed, especially the dragabilly library.

Could you look into that?

**Comment by stvstnfrd** [comment]
End of review.

**Comment by azimpradhan** [comment]
I believe that the diffs are caused by the javascript minifier that is part of your xblock engine, @stvstnfrd. As you can see the some of the one letter variable names are renamed if the is re-minified. Here are three of the four diffs:

https://www.diffchecker.com/kvrdsxkh
https://www.diffchecker.com/cirrjtkq
https://www.diffchecker.com/wy8awlsf

I can go ahead and remove these files from the commit, although ideally the js minifier should not produce a diff if there is no change in the js.

**Comment by stvstnfrd** [comment]
So weird..

>  the js minifier should not produce a diff if there is no change in the js

Agreed. I can't help but assume that the minifier (provided by a 3rd party lib) does, in fact, build deterministically. I wonder if you're using a different (perhaps newer) version of the library with slight changes to the compression algorithm.

Regardless, I follow a general rule of "if I didn't try to touch it, I shouldn't commit it", so thanks for making the change.

**Comment by stvstnfrd** [comment]
+1

**Comment by stvstnfrd** [comment]
After this merges, I'll create a release tag and upload the new version to PyPi.
Then we can bump the version number in `edx-platform`.

**Comment by caesar2164** [comment]
:+1:

**Pull request merged**